Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "signer"
+version = "0.1.0"
+authors = ["Joseph Spadavecchia <joseph@redtrie.com>"]
+edition = "2018"
+
+[lib]
+path = "src/lib.rs"
+
+[dependencies]
+oracle = { path = "./proof-systems/oracle" }
+algebra = { path = "./proof-systems/zexe/algebra", features = [ "pasta" ] }
+mina-curves = { path = "./proof-systems/curves" }
+commitment_dlog = { path = "./proof-systems/dlog/commitment" }
```

Comment:
you should point to a git repo (and a commit), not a submodule

Reply:
Since this will be moved into a crate within `proof-systems` this does not need to be done.